lib/identity_doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -36,10 +69,10 @@ def error_messages
         end
 
         def extra_attributes
-          if true_id_product.present? && true_id_product[:AUTHENTICATION_RESULT].present?
+          if true_id_product&.dig(:AUTHENTICATION_RESULT).present? && true_id_product[:AUTHENTICATION_RESULT].present?
```

Comment:
Don't need the right side of the `&&`

Reply:
Oops! That was supposed to have collapsed.

Reply:
Not totally sure how I managed that...